docs/sql/sql.md

```diff
@@ -0,0 +1,440 @@
+# SQL Support
+
+SpacetimeDB supports two subsets of SQL.
+One for ad hoc queries against the database.
+These can be made through the [cli] or the [http] endpoint.
+Another for subscriptions made through the [websocket] api or [sdk].
+
+## Subscriptions
+
```

Comment:
I think is a good idea to preface this by saying that subscriptions require precise semantics for build-derived data and that our SQL subset is designed to perform well and avoid issues.